sklearn/neighbors/_base.py

```diff
@@ -534,6 +534,10 @@ def kneighbors(self, X=None, n_neighbors=None, return_distance=True):
         """Finds the K-neighbors of a point.
         Returns indices of and distances to the neighbors of each point.
 
+        .. versionchanged:: 0.21
```

Comment:
I feel like we wouldn't need these. Thoughts @adrinjalali ?

Reply:
I agree we do not need the versionchanged for `NotFittedError`.